generators/bdk/java/templates/ext-app/application.yaml.ejs

```diff
@@ -0,0 +1,49 @@
+# BDK Core Configuration
+bdk:
+  host: <%= host %>
+  bot:
+    username: <%= username %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+  app:
+    appId: <%= appId %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+# BDK App Configuration
```

Comment:
Should we generate the bundle.js file too? (https://docs.developers.symphony.com/building-extension-applications-on-symphony/app-authentication/obo-authentication#getting-started)

Reply:
The generated project is for the built-in extension app. Not sure if we want to generate bundle.js 🤔